readme.markdown

```diff
@@ -374,11 +374,24 @@ $ node object.js test/x.js test/y.js
 
 # install
 
-With [npm](https://npmjs.org) do:
+With [npm](https://npmjs.org) proceed through the following steps:
 
+- run npm init and set up your package.json: this tells you what node modules others need when working on your repo
```

Comment:
hmm - this is kind of required knowledge to be able to use any npm package; i'm not sure how valuable it is here.

Reply:
I agree with this

Reply:
People who are new to node or NPM can visit this https://docs.npmjs.com/getting-started/what-is-npm which incredible useful for getting started.